tools/local-setup/reset.sh

```
@@ -0,0 +1,5 @@
+docker compose down --volumes --remove-orphans
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)